src/locales/en/pages.json

```diff
@@ -0,0 +1,233 @@
+            "title": {
+                "seed": "Show Seed Phrase?",
+                "privateKey": "Show Private Key?"
+            }
+        },
+        "seedlogin": {
+            "understandTheRisk": "I understand the risk and have privacy",
+            "pleaseAcceptTerms": "Please Accept Terms",
+            "unlockError": "Try Again. Enter the right password (it has 8 or more characters).",
+            "title": {
+                "seed": "Sign-in to See Seed Phrase",
+                "export": "Sign-in to Export Private Key"
+            }
+        },
+        "seedReveal": {
+            "yourSeedPhrase": "Your Seed Phrase",
+            "revealDescription": "The seed phrase is the only way to restore your wallet. Write it down, verify it and then store it securely.",
+            "iSavedTheSeed": "I saved the seed"
+        }
+    },
+    "settings": {
+        "settings": "Settings",
+        "title": "Default Web3 Wallet",
+        "description": "Set Liquality as the default dapp wallet. Other wallets cannot interact with dapps while this is enabled.",
+        "analytics": "Analytics",
+        "analyticsSub": "Share where you click. No identifying data is collected.",
+        "walletLogs": "Wallet Logs",
+        "walletLogsSub": "The wallet logs contain your public information such as addresses and transactions.",
+        "downloadLogs": "Download Logs",
+        "locale": "Language",
+        "localeSub": "Change Language",
+        "dappConnections": "Dapp Connections",
+        "dappConnectionsSub": "Forget all of the dapps connected.",
+        "done": "Done!",
+        "forgetAllConnections": "Forget all connections",
+        "version": "Version"
+    },
+    "experiments": {
+        "experiments": "Experiments",
+        "multiaccounts": "Multi-Accounts",
+        "multiaccountsDescription": "Allow account management and multiple accounts per network"
+    },
+    "manageAssets": {
+        "assetView": "Asset View",
+        "addCustomToken": "Add Custom Token",
+        "cantFindThisToken": "Can't find this token",
+        "addCustomERC20Tokens": "Add Custom ERC20 tokens",
+        "learnHow": "Learn how"
+    },
+    "customToken": {
+        "selectChain": "Select chain...",
+        "tokenContractAddress": "Token Contract Address",
+        "name": "Name",
+        "tokenSymbol": "Token Symbol",
+        "decimals": "Decimals",
+        "addToken": "Add Token",
+        "addCustomToken": "Add Custom Token"
+    },
+    "wallet": {
+        "assets": "Assets",
+        "activity": "Activity",
+        "nfts": {
+            "zero": "NFTs (0)",
+            "one": "NFTs",
+            "other": "NFTs (%{count})"
+        },
+        "nftActivityDescription": "Once you start using your NFTs you will see the activity here",
+        "nftDescription": "Once you start owning NFTs with accounts in your Liquality wallet you will see them here",
+        "exploreNFT": "Explore NFTs on Polygon, Ethereum and Solana",
```

Comment:
Shouldn’t Arbitrum also be mentioned here?

Reply:
thanks @stephien in this case I just copy the actual content from the wallet texts but if we don't need to mention Arbitrum I think that I can remove it just let me know.